addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -79,9 +79,9 @@ def _get_partner_party_vals(self, partner, role):
         return {
             'partner': partner,
             'party_identification_vals': self._get_partner_party_identification_vals_list(partner),
-            'party_name_vals': [{'name': partner.name}],
+            'party_name_vals': [{'name': partner.display_name}],
             'postal_address_vals': self._get_partner_address_vals(partner),
-            'party_tax_scheme_vals': self._get_partner_party_tax_scheme_vals_list(partner, role),
+            'party_tax_scheme_vals': self._get_partner_party_tax_scheme_vals_list(partner.commercial_partner_id, role),
             'party_legal_entity_vals': self._get_partner_party_legal_entity_vals_list(partner),
```

Comment:
I think we should use the commercial_partner_id there as well (and then we can remove the `commercial_partner = partner.commercial_partner_id` in `_get_partner_party_legal_entity_vals_list`)